README.md

```diff
@@ -2,6 +2,10 @@
 
 The membership attribute service provides an API for managing and retrieving membership attributes associated with a user. 
 
+## Nginx configuration
+
+https://github.com/guardian/membership-attribute-service/blob/update-doc/doc/nginx-setup.md
```

Comment:
This link will break once this branch is deleted. Use a relative link: `[nginx setup](doc/nginx-setup.md)`